test/runnable/test22945.d

```diff
@@ -0,0 +1,40 @@
+module test.runnable.test22945 ;
+
+/*
+REQUIRED_ARGS: -release
+PERMUTE_ARGS:  -check=invariant=on
+*/
+
+// FIXME: https://issues.dlang.org/show_bug.cgi?id=22278
```

Comment:
Why this FIXME?

Reply:
Because it needs fixing 😅. That's not the correct bug reference. I used the in and out bug issue because I hadn't created the current invariant issue at the time to not forget to add it afterwards. I then added fixme to update it to the correct bug issue reference. I just forgot to do it before pushing.

Reply:
done